### ISSUE TITLE
Accessibility Reviews

### ISSUE THREAD

**Issue by mgifford** [patrickkunka/mixitup#255]
Has there been any work done on ensuring that this code is accessible or not?

We're looking at this for a project that needs to meet WCAG 2.0 AA.

Looking at the example here there are a lot of problems. I'd expect to see more use of WAI-ARIA for instance as the content is being dynamically generated. 

The example also is missing labels for input forms and any highlighting on focus. 

This could just be the example though. Would love to get more on this neat code.

**Comment by patrickkunka** [comment]
Hi Mike, thanks for your question.

The examples are intentionally as simple as possible to highlight the minimum requirements for implementation.

MixItUp is not concerned with what your markup looks like, and does not dictate your markup structure or how it is styled - it is essentially an enhancement over the top of an existing (ideally responsive) layout. The only requirement is that your target elements are all adjacent siblings.

You are therefore free to implement whatever aria and role tags you see fit for your project's UI. So I see no issues with a MixItUp-enabled interface meeting WCAG 2.0. Also - the content is not dynamically generated (by default anyway), it is simply hidden and shown. So you could certainly use the built-in callback functions or events to update an `aria-hidden` attribute on your target elements (for example).

I will certainly consider writing some guidelines for accessible integrations as part of our upcoming v3 release, but in the mean time I hope that answers your question.

Thanks,
Patrick

**Comment by mgifford** [comment]
I'd have to look at this more deeply. Generally when content is dynamically changed on the page a screen reader should be alerted aria-live does this quite well - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

aria-sort should also be used to indicate the sort order http://rawgit.com/w3c/aria/master/aria/aria.html#aria-sort

These would be things which would have to be in the JS as far as I can tell. 

That's most of what I can tell you without having more experience with your code base.
